Tests/SwerverTests/RequestSpec/HttpRequestParserSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Quick
+import Nimble
+import Swerver
+import Foundation
+
+class HttpRequestParserSpec: QuickSpec {
+    override func spec() {
+        describe("An HTTP Request Parser") {
+            var httpRequestParser: HttpRequestParser!
```

Comment:
I would try to avoid use of `!` in your code. They are OK in tests, but a definite thing to avoid in production code (with some exception). The use of `!` can cause runtime errors that you cannot catch, causing the tests to abort partway through.